### PR TITLE
test distributed_grids/large_vtu_01: fix test

### PR DESCRIPTION
**Pull request by tamiko**
Let's only write to deallog on rank 0, otherwise the output might get
truncated.

In reference to #13703

### CODE REVIEW

**Comment by tamiko** [comment]
@tjhei *ping*

**Comment by tjhei** [comment]
I don't mind the change, but why is the output truncated?

**Review by tjhei — APPROVED** [review]

File: tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output [hunk 1]
```diff
@@ -7,6 +7,8 @@ DEAL:0::<?xml version="1.0" ?>
 <Piece NumberOfPoints="216" NumberOfCells="64" >
 OK
 
-DEAL:1::n_global_active_cells: 8 n_global_levels: 2
-DEAL:1::OK
```
Comment (tamiko) [inline tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output:11]: @tjhei You had output from two of the four participating MPI ranks. I got output from all of the participating MPI ranks.

Comment (tjhei) [reply tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output:11]: Weird. Did I forget to update my output or is something broken?!

**Comment by tjhei** [comment]
Can you also update the other .output file? You should be able to copy and adjust cell numbers.

File: tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output (hunk 1 above)
Comment (tamiko) [reply tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output:11]: That might have been the case - we do not run all configurations in the CI.

**Comment by tamiko** [comment]
@tjhei 
> Can you also update the other .output file? You should be able to copy and adjust cell numbers.

fixed!

**Review by tjhei — APPROVED** [review]

**Review by kronbichler — APPROVED** [review]

**Pull request merged**